src/sites/image/abload.js

```diff
@@ -8,11 +8,13 @@
 
   $.register({
     rule: {
-      host: /^(www\.)?image(pearl|beryl|crest)\.com$/,
+      host: /^(www\.)?image(pearl|crest)\.com$/,
```

Comment:
Separate this change to another commit and pull request.
Removing a site does not have the same semantic with fixing another site.

So you will have two commits, one for removal which should send to `develop` branch, one for fixing which should send to `master`.